scripts/build

```
@@ -8,13 +8,19 @@ poetry run gen-json-schema --closed --no-metadata -t IdentityCheckCredentialJWTC
 poetry run gen-json-schema --closed --no-metadata -t IdentityCheckCredentialClass $HERE/../v1/linkml-schemas/credentials.yaml > $HERE/../v1/json-schemas/IdentityCheckCredential.json
 poetry run gen-json-schema --closed --no-metadata -t AuthorizationRequestClass $HERE/../v1/linkml-schemas/credentials.yaml > $HERE/../v1/json-schemas/AuthorizationRequest.json
 poetry run gen-json-schema --closed --no-metadata -t PostalAddressClass $HERE/../v1/linkml-schemas/address.yaml > $HERE/../v1/json-schemas/PostalAddress.json
+poetry run gen-json-schema --closed --no-metadata -t PassportDetailsClass $HERE/../v1/linkml-schemas/document.yaml > $HERE/../v1/json-schemas/PassportDetails.json
+poetry run gen-json-schema --closed --no-metadata -t DrivingPermitDetailsClass $HERE/../v1/linkml-schemas/document.yaml > $HERE/../v1/json-schemas/DrivingPermit.json
+poetry run gen-json-schema --closed --no-metadata -t ResidencePermitDetailsClass $HERE/../v1/linkml-schemas/document.yaml > $HERE/../v1/json-schemas/ResidencePermit.json
 
 rm -rf $HERE/../docs/v1
 poetry run gen-markdown $HERE/../v1/linkml-schemas/credentials.yaml -d $HERE/../docs/v1/ --index-file index.md
 mkdir $HERE/../docs/v1/examples
 ln -s ../../v1/linkml-schemas $HERE/../docs/v1/linkml-schemas
 ln -s ../../v1/json-schemas $HERE/../docs/v1/json-schemas
 cp -R $HERE/../v1/examples/PostalAddressClass $HERE/../docs/v1/examples/PostalAddressClass
+cp -R $HERE/../v1/examples/PassportDetailsClass $HERE/../docs/v1/examples/PassportDetailsClass
+cp -R $HERE/../v1/examples/DrivingPermitDetailsClass $HERE/../docs/v1/examples/DrivingPermitDetailsClass
+cp -R $HERE/../v1/examples/ResidencePermitDetailsClass $HERE/../docs/v1/examples/ResidencePermitDetailsClass
```

Potentially we might as well just grab all of the examples, in the same way we're grabbing the schemas?

```suggestion
ln -s ../../v1/examples $HERE/../docs/v1/examples
```

(untested)

---

Yeah I think we have some of the test json files in the examples root at the moment, didn't want to move everything until I had evaluated those, as I think we may want to remove some.

---

Tried to command and got the error `Too many levels of symbolic links:` will need to investigate will look in a future pull request.